Kickstarter-iOS/Views/Cells/PledgeAmountCell.swift

```diff
@@ -40,6 +45,9 @@ final class PledgeAmountCell: UITableViewCell, ValueCell {
 
     self.spacer.widthAnchor.constraint(greaterThanOrEqualToConstant: Styles.grid(3)).isActive = true
 
+    self.amountInputView.textField
+      .addTarget(self, action: #selector(self.amountUpdated(_:)), for: .editingChanged)
```

Comment:
Can we replace `self.amountUpdated` with class name instead `PledgeAmountCell.amountUpdated`? 

This is SwiftFormat's doing unfortunately 😄 If the selector is left without `self` SwiftFormat will add it there, so therefore I think we could be explicit and use the class name instead

Reply:
Will also wait for #719 to merge for this.

Reply:
This is merged 👍